src/components/ui/ClusterEventsToolbar.tsx

```diff
@@ -104,13 +124,18 @@ const ClusterEventsToolbar: React.FC<ClustersListToolbarProps> = ({
     const isChecked = event.target.checked;
 
     switch (value) {
-      case NO_HOSTS:
-        setFilters({
-          ...filters,
-          hosts: [], // clear all selection whenever clicked
-          clusterLevel: false,
-          orphanedHosts: false,
-        });
+      case SELECT_ALL:
+        if (filters.selectAll) {
```

Comment:
nit: Can we use `isChecked` instead? And so flip these if-else blocks.

Reply:
done.